graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH8.kt

```diff
@@ -0,0 +1,45 @@
+package com.anaplan.engineering.azuki.graphs.eacs
+
+import com.anaplan.engineering.azuki.core.runner.Eac
+import com.anaplan.engineering.azuki.core.system.BEH
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphBehaviours
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphFunctionalElements
+import com.anaplan.engineering.azuki.graphs.dsl.GraphScenario
+import com.anaplan.engineering.azuki.graphs.graphA
+
+
+@BEH(GraphBehaviours.CreateDirectedGraph, GraphFunctionalElements.DirectedGraph, """
+    Create a directed graph
+""")
+class BEH8 : GraphScenario() {
+
+    @Eac("A directed graph is a graph that only contains directed edges",
+        """
+            A directed edge is an edge with ordered endpoints
+        """)
```

Comment:
⛏️ Formatting. Choose either plain strings or raw strings for both summary and notes, don't mix-and-match.